routers/api/v1/router_test.go

```diff
@@ -81,3 +96,76 @@ func Test_RegisterRoutes__should_register_required_routes(t *testing.T) {
 		})
 	}
 }
+
+func Test_RegisterRoutes__should_set_up_required_auth_verification(t *testing.T) {
+	restoreVars := testutils.SetEnvVars(map[string]string{
+		"JWT_SECRET": "verysecret",
+	})
+	env := environment.NewEnv(zap.NewNop())
+	restoreVars()
+
+	ctrl := gomock.NewController(t)
+	mockUserService := mock_services.NewMockUserService(ctrl)
+
+	mockUserService.EXPECT().GetUsers(gomock.Any()).AnyTimes()
+	mockUserService.EXPECT().GetUserWithEmail(gomock.Any(), gomock.Any()).AnyTimes()
+	mockUserService.EXPECT().GetUserWithID(gomock.Any(), gomock.Any()).Return(nil, errors.New("service err")).AnyTimes()
+
+	router := NewAPIV1Router(zap.NewNop(), nil, mockUserService, nil, env)
+
+	tests := []struct {
+		route        string
+		method       string
+		minAuthLevel authlevels.AuthLevel
+	}{
+		{
+			route:        "/users/",
+			method:       "GET",
+			minAuthLevel: authlevels.Organizer,
+		},
+		{
+			route:        "/users/verify",
+			method:       "GET",
+			minAuthLevel: authlevels.Applicant,
+		},
+		{
+			route:        "/users/me",
+			method:       "GET",
+			minAuthLevel: authlevels.Applicant,
+		},
+		{
+			route:        "/users/me",
+			method:       "PUT",
+			minAuthLevel: authlevels.Applicant,
+		},
+	}
+
+	for _, tt := range tests {
```

Comment:
What does `tt` represent?

Reply:
test inside a test or something, not quite sure. Another golang convention